cellbrowser_tools/bin/build_release.py

```diff
@@ -39,29 +48,52 @@ def get_data_groups(prefs):
     # log.info('ABOUT TO CREATE ' + str(total_jobs) + ' JOBS')
     groups = []
     for index, (fovid, group) in enumerate(data_grouped):
-        groups.append(group)
+        groups.append(group.to_dict(orient="records"))
+    log.info("Converted groups to lists of dicts")
+
     return groups
 
 
-@task
 def process_fov_row(group, args, prefs):
-    rows = group.to_dict(orient="records")
-    createJobsFromCSV.do_image(args, prefs, rows)
+    rows = group  # .to_dict(orient="records")
+    log.info("STARTING FOV")
+    try:
+        createJobsFromCSV.do_image(args, prefs, rows)
+    except Exception as e:
+        log.error("=============================================")
+        if args.debug:
+            log.error("\n\n" + traceback.format_exc())
+            log.error("=============================================")
+        log.error("\n\n" + str(e) + "\n")
+        log.error("=============================================")
+        raise
+    log.info("COMPLETED FOV")
 
 
 @task
+def process_fov_rows(groups, args, prefs, distributed_executor_address):
+    # Batch process the FOVs
+    with DistributedHandler(distributed_executor_address) as handler:
+        handler.batched_map(
+            process_fov_row,
+            [g for g in groups],
+            [args for g in groups],
+            [prefs for g in groups],
+            batch_size=30,
+        )
+    return "Done"
+
+
 def validate_fov_rows(groups, args, prefs):
     validateProcessedImages.validate_rows(groups, args, prefs)
     return True
 
 
-@task
-def build_feature_data(prefs):
-    validateProcessedImages.build_feature_data(prefs)
+def build_feature_data(prefs, groups):
+    validateProcessedImages.build_feature_data(prefs, groups)
     return True
```

Comment:
Since these are mostly holdouts from when everything was Prefect Flow oriented. Maybe just place these in the main function instead of being written as @task wrappers

Reply:
I'm going to leave these because I am thinking about modifying build_release to run in either of "sbatch" mode or "prefect" mode.